<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialogUi</class>
 <widget class="QWidget" name="SettingDialogUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>置換目印の先頭</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="replace_sign_head"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>置換目印の末尾</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="replace_sign_end"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="BrowseTextboxSettingDialog" name="wizard_dir_line_edit" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="default_button">
       <property name="text">
        <string>デフォルト値に戻す</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BrowseTextboxSettingDialog</class>
   <extends>QWidget</extends>
   <header location="global">browse_textbox_setting_dialog.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
